STEM_Rules.md

```diff
@@ -211,4 +213,4 @@ We will NOT ACCEPT these as responsible justifications and therefore valid reaso
 + Downplaying severity. e.g. "No one was harmed" → Inviting greater consequences to happen before you regret it. We already assigned a punishment appropriate for the severity.
 + Proof by assertion won't do you any favours. The following (non-exhaustive) list of behaviours during an appeal is grounds for immediate appeal denial or even punishment aggravation, at staff discretion: Verbal abuse, spam-pinging, or other contempt of staff.
 + Trolling or otherwise uncooperative behaviour.
-</details>
\ No newline at end of file
+</details>
```

Comment:
i dont know what editor you are using but please dont mess up with the final lines